internal/commands/stack/suggest_stacks.go

```diff
@@ -1,26 +1,27 @@
-package commands
+package stack
```

Comment:
What's the value or reasoning for `stack` to be its own package outside of `commands`? It smells like an anti-pattern but maybe I'm missing something.

Reply:
Based on conversation with the team, we decided to keep it in the `commands` package.